src/components/UrgencyTag.jsx

```diff
@@ -0,0 +1,34 @@
+export const UrgencyTag = ({
+	color,
+	value,
+	frequency,
+	handleFrequencyChange,
+	tagName,
+}) => {
+	return (
+		<div className="h-[120px]">
+			<div
+				className={
+					color +
+					' w-32 h-72  rounded-t-full relative hover:scale-105 mx-auto -rotate-90 sm:rotate-0 border-2 border-gray-400'
+				}
+			>
+				<input
+					id={value}
+					type="radio"
+					name="frequency"
+					value={value}
+					className="top-4 left-[42%] sm:left-[40%]  absolute cursor-pointer"
+					checked={frequency === value}
+					onChange={handleFrequencyChange}
+				/>
+				<label
+					htmlFor={value}
+					className="absolute w-52 sm:w-full bg-white top-[50%] -right-10 sm:right-0 text-center border rotate-90 sm:rotate-0 cursor-pointer"
```

Comment:
Since our app is set to auto-detect light and dark mode, we need to probably add the dark mode styling. Even if we don't add a toggle in the future. As you can see, I can't see the text in dark mode. 
![Screenshot 2023-09-30 at 8 29 34 AM](https://github.com/the-collab-lab/tcl-64-smart-shopping-list/assets/97368143/c6d6816c-120b-4078-9cb4-c99b2a21dcd3). 

Additionally, I think we should set the white labels to a specific height in the desktop view. That way they maintain consistent styling.

Reply:
Good catch. I didn't check the dark mode. Black text and white background would be fine for the dark mode, or should I change them?

Reply:
Added border for the selected tag. It looks like this now.

![Screenshot from 2023-09-30 08-36-58](https://github.com/the-collab-lab/tcl-64-smart-shopping-list/assets/73622805/87225601-fbea-425d-b06a-e070cbb32388)

Reply:
Looks great. Thanks.